src/patternfly/components/DataToolbar/data-toolbar.scss

```diff
@@ -0,0 +1,357 @@
+$pf-c-data-toolbar--breakpoint-map: build-breakpoint-map("base", "md", "lg", "xl", "2xl");
+$pf-c-data-toolbar--spacer-map: build-spacer-map("none", "sm", "md", "lg");
+
+// Data toolbar
+.pf-c-data-toolbar {
+  --pf-c-data-toolbar--BackgroundColor: var(--pf-global--BackgroundColor--100);
+  --pf-c-data-toolbar--GridGap: var(--pf-global--spacer--md);
+
+  // Expandable content
+  --pf-c-data-toolbar__expandable-content--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-data-toolbar__expandable-content--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-data-toolbar__expandable-content--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-data-toolbar__expandable-content--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-data-toolbar__expandable-content--lg--PaddingTop: 0;
+  --pf-c-data-toolbar__expandable-content--lg--PaddingRight: 0;
+  --pf-c-data-toolbar__expandable-content--lg--PaddingBottom: 0;
+  --pf-c-data-toolbar__expandable-content--lg--PaddingLeft: 0;
+  --pf-c-data-toolbar__expandable-content--ZIndex: var(--pf-global--ZIndex--xs);
+  --pf-c-data-toolbar__expandable-content--BoxShadow: var(--pf-global--BoxShadow--md-bottom);
+
+  @media screen and (min-width: $pf-global--breakpoint--lg) {
+    --pf-c-data-toolbar__expandable-content--PaddingTop: var(--pf-c-data-toolbar__expandable-content--lg--PaddingTop);
+    --pf-c-data-toolbar__expandable-content--PaddingRight: var(--pf-c-data-toolbar__expandable-content--lg--PaddingRight);
+    --pf-c-data-toolbar__expandable-content--PaddingBottom: var(--pf-c-data-toolbar__expandable-content--lg--PaddingBottom);
+    --pf-c-data-toolbar__expandable-content--PaddingLeft: var(--pf-c-data-toolbar__expandable-content--lg--PaddingLeft);
+  }
+
+  // Expanded
+  --pf-c-data-toolbar__expandable-content--m-expanded--GridRowGap: var(--pf-global--gutter);
+
+  // Base spacer - shared value
+  --pf-c-data-toolbar--spacer--base: var(--pf-global--spacer--md);
+
+  // Spacer values
+  --pf-c-data-toolbar__item--spacer: var(--pf-c-data-toolbar--spacer--base);
+  --pf-c-data-toolbar__group--spacer: var(--pf-c-data-toolbar--spacer--base);
+
+  // Focus state
+  --pf-c-data-toolbar__item--child--focus--ZIndex: var(--pf-global--ZIndex--xs);
+
+  // Separator
+  --pf-c-data-toolbar--m-separator--spacer: var(--pf-c-data-toolbar__item--spacer);
+  --pf-c-data-toolbar--m-separator--BackgroundColor: var(--pf-global--BorderColor--100);
+  --pf-c-data-toolbar--m-separator--Width: var(--pf-global--BorderWidth--md);
+  --pf-c-data-toolbar--m-separator--Height: var(--pf-global--FontSize--lg);
```

Comment:
This doesn't really follow the var naming convention, except that it follows the selector order in the SCSS. Though ideally in the SCSS, because of how we write BEM modifiers, the modifier should be combined with the selector it applies to, otherwise (since we share modifier class names, and the class name doesn't include the element it applies to) we run into potential conflicts with `.pf-c-data-toolbar .pf-m-separator {}` applying to multiple elements  that share the modifier class name (items, groups, and any other child of the toolbar - dropdowns, etc - that may use that class). And writing the var this way opens us up to conflicting names where, as an example, if we were to add `.pf-m-separator` to `.pf-c-data-toolbar`, the variable name (`--pf-c-data-toolbar--m-separator`) is already taken.

In a perfect world, the `.pf-m-separator` selector would be combined with the element it modifies, so `.pf-c-toolbar__item.pf-m-separator`, then the var name would be `--pf-c-data-toolbar__item--m-separator`. What do you think?

Reply:
@mcoker updated, how's that look?

Reply:
Perfect, thanks!